backend/package.json

```diff
@@ -1,6 +1,19 @@
 {
-  "type": "module",
+  "type": "commonjs",
   "dependencies": {
```

Comment:
Why this change?

Reply:
In the type module we cannot use the exports or module.exports(we have used this this is somewhere in our codebase i have checked but i don't know where we have used that. this is the reason i have useed type="commonjs" @krishnaacharyaa

Reply:
But module is better than commonjs, 

> we have used this is somewhere in our codebase i have checked but i don't know where we have used that
Can you figure out where and rectify by searching ctrl + shift +f   

If not only then we'll change, 
Let's not change because we are getting some error, instead lets see which is best commonjs or module? And then change our codebase

Reply:
In the ts config also they the module is mentioned as common js @krishnaacharyaa

Reply:
@krishnaacharyaa Yeah, module is better than commonjs.

Reply:
> In the type module we cannot use the exports or module.exports(we have used this is somewhere in our codebase i have checked but i don't know where we have used that. this is the reason i have useed type="commonjs" @krishnaacharyaa

Use export syntax instead of module.exports
example:
`
export const replaceStr = (str, char, replacer) => {
  const regex = new RegExp(char, "g")
  const replaced = str.replace(regex, replacer)
  return replaced
}
`

Reply:
@Sukomal07 This is the js code compiled by the ts compiler, In my ts code i did not use the module.exports.